src/btree/bt_handle.c

```diff
@@ -146,8 +146,10 @@ __wt_btree_open(WT_SESSION_IMPL *session, const char *op_cfg[])
             WT_ERR(__wti_btree_tree_open(session, root_addr, root_addr_size));
 
             /* Warm the cache, if possible. */
-            // WT_WITH_PAGE_INDEX(session, ret = __btree_preload(session));
-            // WT_ERR(ret);
+            if (!__wt_conn_is_disagg(session)) {
+                WT_WITH_PAGE_INDEX(session, ret = __btree_preload(session));
```

Comment:
You need to restore this function back (copy from develop). I removed the implementation of it.